pubsub/CHANGES.md

```diff
@@ -1,11 +1,23 @@
 # Changes
 
-### [1.8.3](https://www.github.com/googleapis/google-cloud-go/compare/pubsub/v1.8.2...v1.8.3) (2020-11-10)
+## [1.9.0](https://www.github.com/googleapis/google-cloud-go/compare/pubsub/v1.8.3...v1.9.0) (2020-12-03)
+
+### Features
+
+- **pubsub:** Enable server side flow control by default with the option to turn it off ([#3154](https://www.github.com/googleapis/google-cloud-go/issues/3154)) ([e392e61](https://www.github.com/googleapis/google-cloud-go/commit/e392e6157ee02a344528de63ab16baba61470b24))
+
+### Refactor
 
+**NOTE**: There were several changes that were proposed to be made to the library around exposing `Message` and `PublishResult`. However, the decision was made to only allow packages in `google-cloud-go` to access `NewMessage` and `NewPublishResult` (see #3351).
```

Comment:
Wording nit: "Several changes were proposed for allowing `Message` and `PublishResult` to be used outside the library."